ui/components/MesheryNotification.js

```diff
@@ -122,6 +124,11 @@ class MesheryNotification extends React.Component {
       const { events } = this.state;
       const data = JSON.parse(e.data);
       events.push(data);
+      //Temperory Hack
+      if(data.summary==="Smi conformance test completed successfully"){
```

Comment:
The response should be consumed from the trigger call, this is really not good 😕 .

What you can do is check for the SMI click in adapter play component and rather than having a notification stream for it, just display `Test completed` and stream the results into store

Reply:
Yes you're right. Although this us just a draft. We are phasing out the changes. We'll be removing these eh the pr is ready to merge

Reply:
Yeah. I know. This is not at all the way we should or are going to do it! Notification Component should not be involved in any way.

Reply:
Resolving comment.

Signal when ready to test, @dhruv0000